### ISSUE TITLE
Bugs in Advanced options in cooperation with screen readers

### ISSUE THREAD

**Issue by romw** [BOINC/boinc#1528]
**Reported by krzyszp on 18 Dec 41858584 22:50 UTC**
In "Advanced/Select computer" window labels "Host name" and "Password" are not readable by screen readers

Migrated-From: http://boinc.berkeley.edu/trac/ticket/1151

**Comment by romw** [comment]
**Commented by romw on 16 Apr 41858682 03:28 UTC**
I'm afraid I'm not able to reproduce this issue.

According to the !AccCheck tool:  http://acccheck.codeplex.com/

Both the host name and password controls are exposed to both the MSAA interface and the newer MSUIA interface.
